src/conversions/indexmap.rs

```diff
@@ -0,0 +1,132 @@
+#[cfg(feature = "indexmap")]
+#[cfg_attr(docsrs, doc(cfg(feature = "indexmap")))]
+mod indexmap_indexmap_conversion {
```

Comment:
Let's move these to `conversions/mod.rs` which allows you to drop one level of nesting.

```suggestion
```